lib/plugins/package/lib/packageService.js

```diff
@@ -40,6 +40,11 @@ module.exports = {
       this.serverless.service.plugins
     ).localPath;
     const localPathExcludes = pluginsLocalPath ? [pluginsLocalPath] : [];
+    const localPluginExcludes = this.serverless.pluginManager
+      .getLocalPluginsPaths()
+      .map((pluginPath) => {
+        return pluginPath.endsWith('/') ? path.join(pluginPath, '/**') : pluginPath;
+      });
```

Comment:
As mentioned in previous comments, it'll be great to resolve patterns on spot, and then here simply:

```javascript
const localPathExcludes = Array.from(this.serverless.pluginManager.localPluginPathPatterns)
if (pluginsLocalPath) localPathExcludes .push(pluginsLocalPath)
```